packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -586,7 +586,10 @@ abstract class RenderSliverMultiBoxAdaptor extends RenderSliver
   @override
   void applyPaintTransform(RenderBox child, Matrix4 transform) {
     if (_keepAliveBucket.containsKey(indexOf(child))) {
-      // The children that are kept alive can't be drawn.
+      // It is possible that widgets under kept alive children want to paint
```

Comment:
It is possible that children in the keepAlive list will want to paint themselves.